modules/cnv-monitoring-fencing-events.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-fencing-nodes.adoc
+
+[id="cnv-monitoring-fencing-events_{context}"]
+
+= Monitoring fencing events
```

Comment:
In 4.3 it should be possible to look for k8s events instead.
    `oc get events -n openshift-machine-api --field-selector involvedObject.kind=Machine`
See https://asciinema.org/a/ZYkbsUyXciEwPe1ZUlsifObhK, you can see the full list of events at the 1:40 mark.